compiler-core/src/warning.rs

```diff
@@ -1258,19 +1259,19 @@ doesn't fit in that many {unit}. It would be truncated by taking its {taken}, re
                     }
                 }
 
-                type_::Warning::AssertLiteralValue { location } => Diagnostic {
+                type_::Warning::AssertLiteralBool { location } => Diagnostic {
                     title: "Assertion of a literal value".into(),
                     text: wrap(
-                        "Asserting that a literal value is redundant since you \
-can already tell whether it will be true or false.",
+                        "Asserting on a literal Boolean is redundant since you \
+can already tell whether it will be True or False.",
```

Comment:
```suggestion
can already tell whether it will be `True` or `False`.",
```

If you want to use code then add backticks